core/src/main/java/com/graphhopper/routing/util/EncodedDoubleValue.java

```diff
@@ -31,6 +31,11 @@ public EncodedDoubleValue(String name, int shift, int bits, double factor, long
 
     public EncodedDoubleValue(String name, int shift, int bits, double factor, long defaultValue, int maxValue, boolean allowZero) {
         super(name, shift, bits, factor, defaultValue, maxValue, allowZero);
+        double factorDivision = maxValue / factor;
+        // Interestingly we cannot use modulo here, e.g. 10%00.1 != 0
```

Comment:
This is not unexpected :) http://stackoverflow.com/questions/3227342/modulus-with-doubles-in-java